src/bin/pgcopydb/schema.c

```diff
@@ -423,11 +423,34 @@ schema_list_extensions(PGSQL *pgsql, DatabaseCatalog *catalog)
 	SourceExtensionArrayContext parseContext = { { 0 }, catalog, false };
 
 	char *sql =
+		"with recursive fk_constraints as ( "
+		"     select * from pg_constraint where contype = 'f' and conrelid <> confrelid "
+		" ), raw_ordered_fk_constraints as ( "
+		"     select "
+		"            distinct c.confrelid as relid, "
+		"            0 as depth "
+		"       from fk_constraints c "
+		"      where not exists ( "
+		"            select * from "
```

Comment:
Another target list to expand.

Reply:
Also move the FROM clause to its own line?

Reply:
done